pkg/kubelet/userns_manager.go

```diff
@@ -0,0 +1,476 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"os"
+	"path"
+	"sync"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+const bitsUint32 = 32
+
+type bitArray struct {
+	data       []uint32
+	firstIndex int
+	size       uint32
```

Comment:
I'd use a native `uint` here, I think.

Reply:
To be clear, it was just the `size` member where I'd use a native `uint`. (In my mind, I'm translating this to Rust and reviewing that - someone with more Golang experience here will have a better picture).

Reply:
@sftim We can drop size here in the end, as we ended up doing some other simplifications so it is not needed. Will mark this as solved when we push removing that.

Reply:
Dropped